### ISSUE TITLE
PHP Warning:  preg_replace(): No ending delimiter '~' found 

### ISSUE THREAD

**Issue by fezfez** [crysalead/kahlan#77]
When i generate coverage clover, this error happen.

```
PHP Warning:  preg_replace(): No ending delimiter '~' found in vendor\crysalead\kahlan\src\reporter\coverage\Collector.php on line 290
PHP Stack trace:
PHP   1. {main}() vendor\crysalead\kahlan\bin\kahlan:0
PHP   2. kahlan\cli\Kahlan->run() vendor\crysalead\kahlan\bin\kahlan:44
PHP   3. filter\Filter::on() vendor\crysalead\kahlan\src\cli\Kahlan.php:369
PHP   4. call_user_func_array() vendor\crysalead\filter\src\Filter.php:215
PHP   5. kahlan\cli\Kahlan->kahlan\cli\{closure}() vendor\crysalead\filter\src\Filter.php:215
PHP   6. kahlan\cli\Kahlan->_reporting() vendor\crysalead\kahlan\src\cli\Kahlan.php:363
PHP   7. filter\Filter::on() vendor\crysalead\kahlan\src\cli\Kahlan.php:558
PHP   8. call_user_func_array() vendor\crysalead\filter\src\Filter.php:215
PHP   9. kahlan\cli\Kahlan->{closure:kahlan-config.php:47-59}() vendor\crysalead\filter\src\Filter.php:215
PHP  10. kahlan\reporter\coverage\exporter\Clover::write() kahlan-config.php:57
PHP  11. kahlan\reporter\coverage\exporter\Clover::export() vendor\crysalead\kahlan\src\reporter\coverage\exporter\Clover.php:27
PHP  12. kahlan\reporter\Coverage->export() vendor\crysalead\kahlan\src\reporter\coverage\exporter\Clover.php:62
PHP  13. kahlan\reporter\coverage\Collector->export() vendor\crysalead\kahlan\src\reporter\Coverage.php:141
PHP  14. preg_replace() vendor\crysalead\kahlan\src\reporter\coverage\Collector.php:290
```

```
php -v
PHP 5.5.8 (cli) (built: Jan  8 2014 15:32:25)
Copyright (c) 1997-2013 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2013 Zend Technologies
```

Kahlan 1.1.8

**Comment by jails** [comment]
Please check with the 1.1.9 version.
But with no more information on the code itself I can only make some guess.

**Comment by jails** [comment]
I'm assuming this issue has been solved in 1.1.9, otherwise open this issue back with further details so I can reproduce the issue.

**Comment by fezfez** [comment]
i will try with new version, i give you feed back
